esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -11,7 +11,7 @@
    initial cycle point = 1
    final cycle point = 5
     [[graph]]
-        R1 = """install_cold => configure => process<fast> => compare<fast>
+        R1 = """install_cold => get_esmvaltool => configure => process<fast> => compare<fast>
                 configure => process<medium> => compare<medium>"""
         3/P2 = compare<fast>[-P2] => configure => process<fast> => compare<fast>
         4/P3 = compare<medium>[-P3] => configure => process<medium> => compare<medium>
```

Comment:
'get_esmvaltool =>' needs to be added as the second step for both cycling workflows:

i.e.    
        3/P2 = compare<fast>[-P2] => get_esmvaltool => configure => process<fast> => compare<fast>
        4/P3 = compare<medium>[-P3] => get_esmvaltool => configure => process<medium> => compare<medium>

Reply:
11813cb7c done.